test/e2e/antctl_test.go

```diff
@@ -38,8 +38,8 @@ type cmdAndReturnCode struct {
 func TestAntctl(t *testing.T) {
 	skipIfHasWindowsNodes(t)
 	skipIfNotRequired(t, "mode-irrelevant")
-
-	data, err := setupTest(t)
+	var testNs = randNamespace("test-antctl-")
+	data, err := setupTest(t, testNs)
```

Comment:
perhaps `setupTest` could use `t.Name()` to generate a dedicated name for each test.

```
namespace := randName(tb.Name())
if err := testData.createNamespace(namespace); err != nil {
	return nil, err
}
```
Of course, some handling is needed to lower the cases and replace some unsupported chars.

Reply:
will do the needful